### ISSUE TITLE
The unity package is bloated and contains a lot of non-essential assets

### ISSUE THREAD

**Issue by hesselmonk** [mapbox/mapbox-unity-sdk#1551]
The unity package for Mapbox is very bloated and unwieldy. I understand that there's value to get (new) users up and running quickly. But it would be _much_ better if there were more packages for more granular control.

For example, I have no interest in the AR packages so to get started now I have to download the Mapbox unity package and then delete more than half of the folders. I don't think it is good practice to package third party packages with your (main) release and there's also a random third party folder in there just to fix the line endings?

And then there's tons of example files, fonts and other assets. Is the `CornerPanelRadius_12px_Fill` essential for the core functionality of the plugin? Does it, at some point, reference any of the OpenSans fonts from its font directory? Is the `frameTest` material essential? I guess the answer to those questions is "no", but it takes a lot of time, trail an effort to figure it out.

Don't get me wrong, I _like_ the functionality you provide, but the way it is packaged leaves a lot of room for improvement. I think separating the Examples, AR and Core into separate packages (ideally in a [UPM](https://docs.unity3d.com/Manual/upm-git.html)) would be a huge improvement.

**Comment by brnkhy** [comment]
@hesselmonk I totally agree. I really want to start a cleanup and restructuring effort but you know how it goes, it's never the top priority but becomes more and more difficult each passing day as well.
I think at this point, simplest first step would be namespacing/foldering everything as much as possible. So AR in on place, samples in another, core is separate of course. And if you know what you're doing you can just add core and built your app on it etc.

**Comment by andybak** [comment]
Any comments on UPM support? It's pretty much the standard way to handle Unity packaging now and OpenUPM is really taking off.

**Comment by brnkhy** [comment]
@andybak I think it's unlikely for v2.1.1 at this point.

**Comment by andybak** [comment]
So... About that UPM support?

**Comment by andybak** [comment]
Where is the UPM support documented? It's not in the readme of the dev branch

**Comment by brnkhy** [comment]
@andybak I'm not sure if I understand it correct but v3 is structured as UPM and it's not on main/dev branch yet, it's still under `sdk3` branch as I'm writing this. There'll be a branch switch, it'll be main but haven't done that yet.
